lib/knackhq/client.rb

```diff
@@ -16,13 +16,13 @@ def objects
       hash_request = request
                      .objects
                      .get
-                     .to_h[:objects]
+                     .payload[0].to_h[:objects]
```

Comment:
Is the payload attribute new to the latest Blanket version?

I believe that this code could also be dried up a bit if you're interested.